src/practices/LanguageIndependent/ReadmeCorrectlySetPractice.ts

```diff
@@ -0,0 +1,97 @@
+import { PracticeEvaluationResult, PracticeImpact } from '../../model';
+import { DxPractice } from '../DxPracticeDecorator';
+import { PracticeContext } from '../../contexts/practice/PracticeContext';
+import { PracticeBase } from '../PracticeBase';
+import { ReportDetailType } from '../../reporters/ReporterData';
+import { FixerContext } from '../../contexts/fixer/FixerContext';
+
+@DxPractice({
+  id: 'LanguageIndependent.ReadmeCorrectlySet',
+  name: 'Set README.md Correctly',
+  impact: PracticeImpact.high,
+  suggestion: 'Provide necessary sections in README',
+  reportOnlyOnce: false,
+  url: 'https://github.com/DXHeroes/knowledge-base-content/blob/master/practices/readme.md',
+  dependsOn: { practicing: ['LanguageIndependent.ReadmeIsPresent'] },
+})
+export class ReadmeIsCorrectlySet extends PracticeBase {
+  private parsedReadme: string[] = [];
+  private readmePath = 'Readme.md';
+
+  async isApplicable(): Promise<boolean> {
+    return true;
+  }
+
+  async evaluate(ctx: PracticeContext): Promise<PracticeEvaluationResult> {
+    if (!ctx.root.fileInspector) {
+      return PracticeEvaluationResult.unknown;
+    }
+
+    const parseReadme = (readmeFile: string) => {
+      return readmeFile
+        .toString()
+        .split(/\r?\n/)
+        .filter((content) => content.trim() !== '');
+    };
+
+    const regexReadme = new RegExp('readme', 'i');
+    const rootFiles = await ctx.root.fileInspector.scanFor(regexReadme, '/', { shallow: true });
+    const filePath = rootFiles[0].path;
+    this.readmePath = filePath;
+    const content = await ctx.root.fileInspector.readFile(filePath);
+
+    const parsedReadme = parseReadme(content);
+    this.parsedReadme = parsedReadme;
+
+    const heading = parsedReadme.filter((value: string) => /^(?:<h1>|#\s+)/.test(value)).length;
```

Comment:
I don't think we should check the level of the sections

Reply:
As per the suggestion for practice, there need to be just 1 h1 heading so this was checking that by filtering and then finding the length.

Reply:
Ok, you're right in this case but I meant it in the whole checking. Like whether the `Getting\s+Started` is there, but not also the `##` if you know what I mean.

Reply:
Ohh i see so the headers can be in h2 or h3 thats what you meant i guess 🤔. I just went with the specs for the issue we can change that and just ensure its not h1 that will be better i guess.

Reply:
Yes! @king-11